src/styles/Navbar.css

```diff
@@ -85,6 +85,18 @@
 	fill: var(--bgBlack);
 }
 
+/* Apply Button */
+a.apply-active {
+	color: white;
+	background-color: var(--lightPurple);
+}
+
+a.apply-inactive {
+	pointer-events: none;
+	color: var(--gray);
+	background-color: var(--bgGray);
+}
+
```

Comment:
Please address the issues I raised previously. To recap:

- The spacing between the Apply button and the rest of the headers doesn't seem even. Please add some more spacing between the button and the rightmost Gallery button.
- When the button is active, let's make the background color the Hack accent color (the reddish pinkish color). `var(--hackAccent)`